apps/src/lib/node/ledger/ethereum_node/oracle/config.rs

```diff
@@ -0,0 +1,26 @@
+//! Configuration for an oracle.
+use namada::types::ethereum_events::EthAddress;
+
+/// Configuration for an oracle.
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
+pub struct Config {
+    /// The minimum number of block confirmations an Ethereum block must have
+    /// before it will be checked for bridge events.
+    pub min_confirmations: u64,
+    /// The Ethereum address of the current bridge contract.
+    pub mint_contract: EthAddress,
+    /// The Ethereum address of the current governance contract.
+    pub governance_contract: EthAddress,
+}
```

Comment:
I still think these parameters should be configured at genesis, and not through ledger config files

Reply:
They will be, this is just the struct to hold them for an `Oracle`. So these fields will be populated by the values from genesis/blockchain storage (though that isn't happening just yet in this PR)

Reply:
I have refactored these params as part of #712.